### DISCUSSION TITLE
Keyboard shortcut to enable/disable window margins

### DISCUSSION THREAD

**Discussion by adammfurman** [ianyh/Amethyst#1638] [Ideas]
As someone who uses a laptop as my main workstation, I am often switching between my home multi-monitor setup and just my laptop. When switching between setups, I manually change the window margins via the GUI as I prefer larger margins when using larger external monitors and smaller margins when on just my laptop to maximize screen-estate.

Requested Solution:
Ability to change or toggle window margins via keyboard shortcuts.

**Comment by sameerasw** [comment]
I too need this... or the smart mode to not remove gaps for single window so in fullscreen mode, I can fill the window without gaps...
